Source/ACE.Entity/Enum/Properties/PropertyInt.cs

```diff
@@ -4,96 +4,87 @@
 
 namespace ACE.Entity.Enum.Properties
 {
+    // No properties are sent to the client unless they featured an attribute.
+    // SendOnLogin gets sent to players in the PlayerDescription event
+    // AssessmentProperty gets sent in successful appraisal
     public enum PropertyInt : ushort
     {
-        // properties marked as ServerOnly are properties we never saw in PCAPs, from here:
-        // http://ac.yotesfan.com/ace_object/not_used_enums.php
-        // source: @OptimShi
-        // description attributes are used by the weenie editor for a cleaner display name
-
         Undef                                    = 0,
-        [ServerOnly]
         ItemType                                 = 1,
+        [AssessmentProperty]
         CreatureType                             = 2,
-        [ServerOnly]
         PaletteTemplate                          = 3,
         ClothingPriority                         = 4,
-        [SendOnLogin]
         EncumbranceVal                           = 5, // ENCUMB_VAL_INT,
```

Comment:
This one should probably be an [AssessmentProperty]. Without it, burden shows as "Unknown" in client for all items

Reply:
Thank you, nice catch!